src/main/java/lotto/Wallet.java

```diff
@@ -0,0 +1,41 @@
+package lotto;
+
+public class Wallet {
+    static final int PRICE = 1000;
+
+    private int budget;
+    private int change;
+
+    public Wallet(int budget) {
+        this.budget = budget;
+        this.change = budget;
+        validate();
+    }
+
+    public boolean isBuyable(int count) {
+        return change > count * PRICE;
```

Comment:
```suggestion
        return change >= count * PRICE;
```
이렇게 수정되는 것이 맞는것이지요..? ㅎ;